functions/analytics.js

```diff
@@ -231,25 +209,21 @@ interface UsersWithVisitsResult {
 
     const csvFormattedLine = dto.map((s) => `"${s}"`).join(",");
 
-    fs.writeFileSync("./dataReport.csv", `${csvFormattedLine} \n`, {
+    fs.writeFileSync(dataReportFileName, `${csvFormattedLine} \n`, {
       flag: "a",
     });
 
-    const result = formattedVisitColumns.map((visit) => visit.timeValue ?? 0);
+    const result = formattedVisitColumns.map((visit) => visit.timeValue || 0);
 
     return result;
   });
 
   // space between visit data & total data
   [0, 1, 2].map(() =>
-    fs.writeFileSync("./dataReport.csv", `\n`, { flag: "a" })
+    fs.writeFileSync(dataReportFileName, `\n`, { flag: "a" })
   );
 
-  const arrayOfResults: {
-    totalAmount: number;
-    totalValue: number;
-    totalUnique: number;
-  }[] = [];
+  const arrayOfResults = [];
```

Comment:
Is this used anywhere?

Reply:
Yep, right below. Note that I didn't write the script itself, just copied it